client/core/core.go

```diff
@@ -3534,6 +3545,59 @@ func (c *Core) GetDEXConfig(dexAddr string, certI interface{}) (*Exchange, error
 	return dc.exchangeInfo(), nil
 }
 
+// AddDEX configures a DEX host connection without creating an account. This
+// allows watching trade activity without registering with the DEX. Register
+// may be used to set up a trading account for this DEX if and when necessary.
+func (c *Core) AddDEX(dexAddr string, certI interface{}) error {
+	if !c.IsInitialized() {
+		return fmt.Errorf("cannot register DEX because app has not been initialized")
+	}
+
+	host, err := addrHost(dexAddr)
+	if err != nil {
+		return newError(addressParseErr, "error parsing address: %w", err)
+	}
+
+	cert, err := parseCert(host, certI, c.net)
+	if err != nil {
+		return newError(fileReadErr, "failed to parse certificate: %w", err)
+	}
+
+	c.connMtx.RLock()
+	_, found := c.conns[host]
+	c.connMtx.RUnlock()
+	if found {
+		return newError(dupeDEXErr, "already connected to DEX at %s", dexAddr)
+	}
+
+	dc, err := c.connectDEX(&db.AccountInfo{
+		Host: host,
+		Cert: cert,
+	})
+	if err != nil {
+		if dc != nil {
+			// Stop (re)connect loop, which may be running even if err != nil.
+			dc.connMaster.Disconnect()
+		}
+		return codedError(connectionErr, err)
+	}
+
+	// Don't allow adding another dex with the same pubKey. There can only be
+	// one dex connection per pubKey. UpdateDEXHost must be called to connect to
+	// the same dex using a different host name.
+	exists, host := c.dexWithPubKeyExists(dc.acct.dexPubKey)
+	if exists {
+		dc.connMaster.Disconnect()
+		return newError(dupeDEXErr, "already connected to DEX at %s but with different host name %s", dexAddr, host)
+	}
+
```

Comment:
Ohhhh. We're not storing the watch-only account in the DB. So if the user had a registered account on server A, and wanted to also see server B view-only, they would need to go to the settings page and add it after starting dexc every time. Hmm.

Reply:
Haven't actually tested this PR yet, but that would be less than ideal UX.
Two options:
- remember watch-only in DB
- use a cookie to remember the desired view-only servers, and they get re-added on-demand

Reply:
Didn't like the idea of storing watch-only account in DB because once added, how's it removed? Although, disable account can be used for that perhaps? Will look into adding to DB, but first, since legacy reg will likely remain past 0.6, I will update the relevant parts of this to consider both legacy and bond payments.

Reply:
> Will look into adding to DB,

Would it be messy to have the frontend re-add the view-only hosts after login, using cookies to remember which view-only hosts were previously added?

Reply:
Maybe not. Was thinking db since it's likely lesser work. But will look into using cookies.

Reply:
That's fine.

I'm also pondering if we eventually want to take this view-only idea one step further and not even require app initialization.  If you just download the client to check out the markets and it wants you to start setting passwords you might walk away.   Obviously that would mean not adding the view-only DEX host to a DB.  Completely stateless on the backend.

Reply:
> I'm also pondering if we eventually want to take this view-only idea one step further and not even require app initialization. If you just download the client to check out the markets and it wants you to start setting passwords you might walk away. Obviously that would mean not adding the view-only DEX host to a DB. Completely stateless on the backend.

That's f****** brilliant!

Reply:
> Obviously that would mean not adding the view-only DEX host to a DB. Completely stateless on the backend.

Does it?

Reply:
Oh, right.  `InitializeClient` is just for the credentials.  Just constructing `Core` with `core.New` does indeed ready the DB file.  Alright then.

Reply:
I'll start with saving to DB. Not requiring init for view-only connection will follow.